### ISSUE TITLE
easy_install tables

### ISSUE THREAD

**Issue by schwancr** [msmbuilder/msmbuilder-legacy#144]
I cannot easy_install pytables, but I can easy_install tables

tables gives you pytables 2.4, whereas pytables gives you v2.3.1

I have tried on my air, my mac mini, and vsp-compute (linux) to no avail. It always fails with the same message:

vsp-compute-01:~ schwancr$ easy_install tables
Searching for tables
Reading http://pypi.python.org/simple/tables/
Reading http://www.pytables.org/
Best match: tables 2.4.0
Downloading http://sourceforge.net/projects/pytables/files/pytables/2.4.0/tables-2.4.0.tar.gz
Processing tables-2.4.0.tar.gz
Writing /tmp/easy_install-c0jctk/tables-2.4.0/setup.cfg
Running tables-2.4.0/setup.py -q bdist_egg --dist-dir /tmp/easy_install-c0jctk/tables-2.4.0/egg-dist-tmp-CKi2Uh
- Found numpy 1.6.1 package installed.
- Found numexpr 2.0.1 package installed.
- Found Cython 0.16 package installed.
  /usr/bin/ld: cannot find -lhdf5
  collect2: ld returned 1 exit status
  /usr/bin/ld: cannot find -lhdf5
  collect2: ld returned 1 exit status
  .. ERROR:: Could not find a local HDF5 installation.
  You may need to explicitly state where your local HDF5 headers and
  library can be found by setting the `HDF5_DIR` environment
  variable or by using the `--hdf5` command-line option.
  error: Setup script exited with 1

Basically I'm missing a C library. If we want people to use v2.4 then we need to have this as a dependency. The other option is to list pytables (rather than tables) as a dependency as this downloads 2.3.1 and can compile.
